projects/bucket-farm/index.js

```diff
@@ -36,8 +36,9 @@ async function tvl(api) {
     let assetType = type.slice(type.indexOf("<"), type.indexOf(">")).split(",")[1].trim()
 
     if(assetType == "0x922d15d7f55c13fd790f6e54397470ec592caa2b508df292a2e8553f3d3b274f::msui::MSUI" || assetType == "0x41ff228bfd566f0c707173ee6413962a77e3929588d010250e4e76f0d1cc0ad4::ksui::KSUI") assetType = ADDRESSES.sui.SUI
-
-    api.add(assetType, degenPool.fields.balance)
+    const credits = degenPool.fields.sheet.fields.credits.fields.contents.reduce((acc,content)=> acc + Number(content.fields.value.fields.pos0),0)
```

Comment:
hi @Jarekkkkk what is this credit field

Reply:
it's the borrowed value from other protocols

Reply:
sorry, you need to expand on that. It is not like other farms where people lock tokens and wait for points/tokens as reward?

Reply:
no, this is some feature we might be utilized on in the future that might integrate with our protocol. if the borrowed value can't be counted, then it's fine. 

thanks for your reply, sir

Reply:
yeah, sorry we cant count borrowed value